### DISCUSSION TITLE
google pay button is not showing in payment

### DISCUSSION THREAD

**Discussion by anbarasannethaji** [flutter-stripe/flutter_stripe#1815] [Q&A]
`  Future<void> initPaymentSheet() async {
    try {
      var paymentDetails = await fetchPaymentDetails();
      Stripe.merchantIdentifier = 'merchant.com.merchant.merchant';
      Stripe.publishableKey = paymentDetails['data']['publishable_key'];
      await Stripe.instance.applySettings();
      await Stripe.instance.initPaymentSheet(
        paymentSheetParameters: SetupPaymentSheetParameters(
          customFlow: false,
          merchantDisplayName: 'merchant name',
          paymentIntentClientSecret: paymentDetails['data']['payment_intent'],
          customerEphemeralKeySecret: paymentDetails['data']['ephemeral_key'],
          customerId: paymentDetails['data']['customer'],
          googlePay: const PaymentSheetGooglePay(
            merchantCountryCode: 'GB',
            currencyCode: 'GBP',
            testEnv: true,
          ),
          applePay: const PaymentSheetApplePay(
            merchantCountryCode: "GB",
            buttonType: PlatformButtonType.buy,
          ),
          style: ThemeMode.dark,
        ),
      );
      setState(() {
        _ready = true;
      });
    } catch (e) {
      setState(() {
        _ready = false;
      });
    }
  }.  `
i have enter this line in android manifest

`  <meta-data
                android:name="com.google.android.gms.wallet.api.enabled" 
                android:value="true" /> `

between application and this is in app level in dependencies

`implementation("com.google.android.gms:play-services-wallet:19.4.0")`

**Comment by debasmitasarkar** [comment]
I have the same issue.

**Comment by nayanbabariya** [comment]
@anbarasannethaji @debasmitasarkar Have you solved it or still have the issue? I am also facing the same.
